Sources/StreamChat/WebSocketClient/EventMiddlewares/UserUpdateMiddleware.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The middleware listens for `UserUpdatedEvent`s and updates `CurrentChatUserDTO` accordingly.
```

Comment:
Yes, this one seems to be a type in the docs, right? @polqf

Reply:
We are talking about that it should be `ChatUserDTO` and not `**Current**ChatUserDTO`, since this event will update any user, and not only the current logged in user